src/packages/@ncigdc/components/Modals/AppendSetModal.js

```diff
@@ -147,7 +147,7 @@ const AppendSetModal = ({
               );
               if (type === 'case') {
                 await analyses
-                  .filter(analysis => analysis.sets.case[selected])
+                  .filter(analysis => analysis.sets.case && analysis.sets.case[selected])
```

Comment:
Added code guard to compensate for non-clinical analyses. Removes console error.